tests/test_wrappers.py

```diff
@@ -0,0 +1,251 @@
+    # transformed output
+    assert all(dataframe_na_transformed[na_variables_left_after_imputation].isna().sum() != 0)
+    assert all(dataframe_na_transformed[variables_to_impute].isna().sum() == 0)
+    pd.testing.assert_frame_equal(ref, dataframe_na_transformed)
+
+
+def test_sklearn_imputer_object_with_constant(dataframe_na):
+    variables_to_impute = ['Name', 'City']
+    na_variables_left_after_imputation = [col
+                                          for col
+                                          in dataframe_na.loc[:, dataframe_na.isna().any()].columns
+                                          if col not in variables_to_impute]
+
+    transformer = SklearnTransformerWrapper(transformer=SimpleImputer(fill_value='missing', strategy='constant'),
+                                            variables=variables_to_impute)
+
+    # transformed dataframe
+    ref = dataframe_na.copy()
+    ref[variables_to_impute] = ref[variables_to_impute].fillna('missing')
+
+    dataframe_na_transformed = transformer.fit_transform(dataframe_na)
+
+    # init params
+    assert isinstance(transformer.transformer, SimpleImputer)
+    assert transformer.variables == variables_to_impute
+    # fit params
+    assert transformer.input_shape_ == (8, 6)
+    # transformed output
+    assert all(dataframe_na_transformed[na_variables_left_after_imputation].isna().sum() != 0)
+    assert all(dataframe_na_transformed[variables_to_impute].isna().sum() == 0)
+    pd.testing.assert_frame_equal(ref, dataframe_na_transformed)
+
+
+def test_sklearn_imputer_allfeatures_with_constant(dataframe_na):
+    transformer = SklearnTransformerWrapper(transformer=SimpleImputer(fill_value='missing', strategy='constant'))
+
+    # transformed dataframe
+    ref = dataframe_na.copy()
+    ref = ref.fillna('missing')
+
+    dataframe_na_transformed = transformer.fit_transform(dataframe_na)
+
+    # init params
+    assert isinstance(transformer.transformer, SimpleImputer)
+    # fit params
+    assert transformer.input_shape_ == (8, 6)
+    # transformed output
+    assert all(dataframe_na_transformed.isna().sum() == 0)
+    pd.testing.assert_frame_equal(ref, dataframe_na_transformed)
+
+
+def test_sklearn_standardscaler_numeric(dataframe_vartypes):
+    variables_to_scaling = ['Age', 'Marks']
+    transformer = SklearnTransformerWrapper(transformer=StandardScaler(), variables=variables_to_scaling)
+
+    ref = dataframe_vartypes.copy()
+    ref[variables_to_scaling] = (ref[variables_to_scaling] - ref[variables_to_scaling].mean()) / ref[
+        variables_to_scaling].std(ddof=0)
+
+    transformed_df = transformer.fit_transform(dataframe_vartypes)
+
+    # init params
+    assert isinstance(transformer.transformer, StandardScaler)
+    assert transformer.variables == variables_to_scaling
+    # fit params
+    assert transformer.input_shape_ == (4, 5)
+    assert (transformer.transformer.mean_.round(6) == np.array([19.5, 0.75]).round(6)).all()
```

Comment:
is 19.5 and 0.75 the mean of Age and Marks? if they have 2 decimals, why round 6?

Reply:
Yes, they are means of Age and Marks. Good point, round here is not necessary at all. Will change this in next commit.